plyer/platforms/linux/audio.py

```diff
@@ -0,0 +1,27 @@
+from plyer.facades.audio import Audio
+import subprocess
+
+
+class LinuxAudio(Audio):
```

Comment:
This is definitely a GStreamer used as a provider for audio, not a "Linux" audio, so it'd nice to name it appropriately. There are other providers and I believe gnome or [kde](https://en.wikipedia.org/wiki/Amarok_(software)) has even its own minimalistic one.